src/main/scala/mirroring/services/SqlService.scala

```diff
@@ -66,6 +66,12 @@ object SqlService extends LogSupport {
           config.logRetentionDuration,
           config.deletedFileRetentionDuration
         )
+        if (config.isChangeTrackingEnabled) {
```

Comment:
it's already set in ChangeTrackingService, why set it a second time?

Reply:
Weirdly enough the commit message is ignored for the second commit. That's why this behaviour has been added in b96ffaeff2c3da97f3b9a75c4513b3799491860e